app/views/sessions/new.html.haml

```diff
@@ -0,0 +1,15 @@
+%section.sign_in.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = bootstrap_form_tag(url: sign_in_path, class: "sign_in") do |f|
+        %header
+          %h1 Sign in
+        %fieldset
+          .form-group.row
+            .col-sm-4
+              = f.email_field :email, class: "form-control"
```

Comment:
I could be wrong but I believe the `form-control` class will be applied by `bootstrap_form_tag` automatically.